bad_code.coffee

```diff
@@ -0,0 +1,4 @@
+class Gangster
+
+    wasItAGoodDay : () ->
```

Comment:
Line contains inconsistent indentation<br>Colon assignment without proper spacing<br>Empty parameter list is forbidden